src/aap_eda/settings/default.py

```diff
@@ -230,7 +234,8 @@ def _get_secret_key() -> str:
     "DEFAULT_PAGINATION_CLASS": "aap_eda.api.pagination.DefaultPagination",
     "PAGE_SIZE": 20,
     "DEFAULT_AUTHENTICATION_CLASSES": [
-        "aap_eda.api.authentication.SessionAuthentication",
+        # LDAP Integration for 2.4 https://issues.redhat.com/browse/AAP-16938
+        "ansible_base.authentication.session.SessionAuthentication",
```

Comment:
@Dostonbek1 does this resolve the issue that the UI had found https://github.com/ansible/eda-server/blob/0d70bad6e523ceb680c3f23c91ddf77a5dc8b966/src/aap_eda/api/authentication.py#L9 is this pattern consistent with what has been implemented for controller.

Reply:
This doesn't change anything functionally. It is a step towards the main goal of having ansible-base which is to export all common pieces of AWX, Hub and EDA to have more consistent code across all AAP components. We might not even need this change for this LDAP stuff so we could remove it for now to avoid potential issues in the future.

Reply:
@Dostonbek1 https://github.com/ansible/django-ansible-base/blob/9c92bf26eaa205fd74269c1d7b47369f156b981b/ansible_base/authentication/session.py#L9 seems to be preserving the 401 so this is 👍